nequip/train/trainer.py

```diff
@@ -213,6 +213,7 @@ def __init__(
         self,
         model,
         model_builders: Optional[list] = [],
+        device: Optional[str] = "cuda" if torch.cuda.is_available() else "cpu",
```

Comment:
why `Optional`?

Reply:
yeah. shouldn't be optional. Removed the optional type